lib/parameters_strapi.py

```diff
@@ -0,0 +1,273 @@
+###
+### Copyright (C) 2024 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+import copy
+import itertools
+
+def gen_avc_cqp_strapi_variants(spec, profiles):
```

Comment:
I think I pointed it out before...
But, how about we consolidate the parameters from all rcmode into a single generator? It would look similar to `gen_avc_forced_idr_variants`.  It would reduce a lot of duplication and lines of code.  So, perhaps the test signature/keys for all avc and hevc could be `("case", "rcmode", "gop", "slices", "bframes", "bitrate", "qp", "quality", "fps", "lowpower", "profile")` where their values are `None` when they don't make sense for the given rcmode.  For av1, remove slices and add tilecols and tilerows?

Reply:
used the consolidated generators, so there is no this issue